diff --git a/./Checkstyle.java b/./GoogleFormatter.java
index 493a1f9..1d240d2 100644
--- a/./Checkstyle.java
+++ b/./GoogleFormatter.java
@@ -1,7 +1,7 @@
 package com.examples.nonemptyblocks;

 /** testing. */
-public class Checkstyle {
+public class GoogleFormatter {
   /** testing. */
   public void method() {
     {
@@ -21,11 +21,12 @@ public class Checkstyle {
     int no = 23;

     switch (no) {
-      case 2: {
-        int x = foo();
-        frob(x);
-        break;
-      }
+      case 2:
+        {
+          int x = foo();
+          frob(x);
+          break;
+        }

       default:
         frob(no);
@@ -36,5 +37,7 @@ public class Checkstyle {
     }
   }

-  public int getX() { return 0; }
+  public int getX() {
+    return 0;
+  }
 }
